ui/src/app/base/components/TitledSection/TitledSection.test.tsx

```diff
@@ -38,3 +48,33 @@ it("can display buttons", () => {
   );
   expect(screen.getAllByRole("button").length).toBe(2);
 });
+
+it("displays a custom heading", () => {
```

Comment:
I'm not sure this tests what you're expecting. The two props `headingElement` and `headingVisuallyHidden` could be removed from this test and it would still pass, in other words, the heading could be visible and appear as something other than a h4 and it would still pass.

Reply:
Good catch, updated.